graylog2-web-interface/src/views/components/views/ExecutionInfo.tsx

```diff
@@ -24,12 +24,13 @@ import { Timestamp } from 'components/common';
 
 const ExecutionInfo = () => {
   const result = useAppSelector(selectCurrentQueryResults);
-  const total = result?.searchTypes && Object.values(result?.searchTypes)?.[0]?.total;
 
   if (isEmpty(result)) {
     return <i>No query executed yet.</i>;
   }
 
+  const total = result?.searchTypes && Object.values(result?.searchTypes)?.find((e) => e.total)?.total;
```

Comment:
Since now we have at least 2 widgets where total value doesn't relate to search results, I think would be nice to have some kind of filtration by widget type, and only then try to pick the total value.


```
const SEARCH_TYPES = ['pivot', 'messages', 'logs']
const total = result?.searchTypes && Object.values(result?.searchTypes)?.find((e) => e.total && SEARCH_TYPES.includes(e.type))?.total;
```

Reply:
I think Jan's approach is fine. We should couple it to the result structure instead of the search type's type, otherwise we would have to extend it every time we add a new search type which provides a total count.